docs/tutorials/pre_executed/timeseries.ipynb

```diff
@@ -0,0 +1,1471 @@
+{
```

Comment:
<p>Re-run with more memory, so no warning?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/834/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>I think that was a consequence of running on a slightly underpowered machine. No memory warnings on the latest run on my MacBook Pro.</p>